packages/edgebridge/README.md

```diff
@@ -0,0 +1,142 @@
+
+# React Native Adobe Experience Platform Edge Bridge extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepedgebridge.svg)](https://www.npmjs.com/package/@adobe/react-native-aepedgebridge)
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepedgebridge)](https://www.npmjs.com/package/@adobe/react-native-aepedgebridge)
+
+`@adobe/react-native-aepedgebridge` is a wrapper around the iOS and Android Adobe Experience Platform Edge Bridge to allow for integration with React Native applications.
+
+The Adobe Experience Platform Edge Bridge mobile extension enables forwarding of Analytics track events to Experience Platform Edge Network when using the [Experience Platform Mobile SDK](https://developer.adobe.com/client-sdks) and the Edge Network extension. The complete ingestion process occurs in two main steps:
+
+1. Edge Bridge forwards the track event data to Edge Network in a generic data format. 
+2. The generic data is converted into the Experience Data Model (XDM) format, which is a standard experience-driven data schema for Adobe and partner solutions. 
+   * This conversion mapping can be set in the Data Collection datastream associated with the application's mobile property using [Data Prep for Data Collection](https://experienceleague.adobe.com/docs/platform-learn/data-collection/edge-network/data-prep.html).
+  
+   * For more info, refer to [Data Prep for Data Collection Documentation](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html).
```

Comment:
Can we summarize this in a shorter content, similar to iOS/Android:

The Edge Bridge mobile extension provides seamless routing of data to the Adobe Experience Platform Edge Network for existing SDK implementations. For applications which already make use of the [MobileCore.trackAction](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#trackaction) and/or [MobileCore.trackState](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#trackstate) APIs to send data to Adobe Analytics, this extension will automatically route those API calls to the Edge Network, making the data available for mapping to a user's XDM schema using the [Data Prep for Data Collection](https://experienceleague.adobe.com/docs/experience-platform/data-prep/home.html).